collection-scripts/collect_service_logs

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
This implementation doesn't take into account how how scripts like https://github.com/openshift/must-gather/blob/b308b60f9501006342e1fdd918961cf0ae9a4f96/collection-scripts/gather_network_logs#L221 supply limited like node lists (to run the collection on). 

This could confuse users, as host logs could be collected from all nodes, while only some pod based operations are only run on specific nodes. I think we need to explore how to account for this.

Reply:
Good point! this is fixed now.
I tested this with: `oc adm must-gather --image=quay.io/jdesousa/must-gather:journalovs gather_network_logs ci-ln-g7g54tb-f76d1-lrzkc-master-1 ci-ln-g7g54tb-f76d1-lrzkc-master-2` and just acquired  it for these two nodes.